components/footer/footer.css

```diff
@@ -0,0 +1,20 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer__title {
```

Comment:
I read some articles about BEM and there are some examples that separate files should be created to collect css rules for every block element. Should it have beed done?

Reply:
No, as I mentioned in previous lesson - we are not going to follow all these rules, we gonna use BEM only for class-naming.